coderd/database/migrations/000257_workspace_agent_script_timings.up.sql

```diff
@@ -0,0 +1,26 @@
+ALTER TABLE workspace_agent_scripts ADD COLUMN id uuid UNIQUE NOT NULL DEFAULT gen_random_uuid();
+
+CREATE TYPE workspace_agent_script_timing_stage AS ENUM (
+    'start',
+    'stop',
+    'cron'
+    );
```

Comment:
```suggestion
);
COMMENT ON TYPE workspace_agent_script_timing_stage IS 'Some explanation...';   
```
Sorry I missed this in my last review, but I always try to encourage everyone to add comments to types, tables and columns. Not a blocker, but would be nice.

Reply:
Done!